### DISCUSSION TITLE
Does this library support SSR?

### DISCUSSION THREAD

**Discussion by haveamission** [jpudysz/react-native-unistyles#48] [Q&A] [answered]
Does this library support SSR at all for NextJS?

**Comment by jpudysz** [comment]
> Does this library support SSR at all for NextJS?

To be honest I don't know. I haven't try it with SSR. I will create simple [solito](https://solito.dev) repo and share my findings.

**Reply by jpudysz** [reply]
I was able to run it, however a few changes in the code were necessary.
I will support it soon in v.1.1

https://github.com/jpudysz/react-native-unistyles/assets/9088288/243b9927-cd38-41a3-a390-ad4233aef160

**Comment by jpudysz** [comment, marked as answer]
Released with #50
